features/Kubernetes_MCS.md

```diff
@@ -69,7 +70,8 @@ Add support for [Kubernetes Multi-cluster Services (MCS)](https://github.com/kub
 [//]: # (for building running a `istioctl experimental command`, or using the preview profile,)
 [//]: # (or any other relevant information.)
 
-- [ ] [Feedback plan](insert_your_link_here).
+- [ ] Feedback plan
+   - Github issues
```

Comment:
is there some time frame that you want to address this in? I'll leave it up to the work groups to decide.. just thinking something like:

 will watch for GitHub issues for two releases in order to decide if this feature is useful and how it might be configured